# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/pagination/fieldset-006.html
# Commit: 0186165ee1d8
# Full Hash: 0186165ee1d84f56362ba16bbda4eefa1d47fd60
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2019-11-26 05:19:55
# Regressor Bug: 471015
# File Overlap Count: 1
# Description:
#   Bug 471015 - [css-break] Implement <fieldset> fragmentation.  r=TYLin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D53709
# ==============================================================================

diff -r 9cf1851c76dc -r 0186165ee1d8 layout/reftests/pagination/fieldset-006.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/layout/reftests/pagination/fieldset-006.html	Mon Nov 25 18:05:16 2019 +0000
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html class="reftest-paged">
+<head>
+  <title>CSS Test: fieldset fragmentation</title>
+  <link rel="author" title="Mats Palmgren" href="https://bugzilla.mozilla.org/show_bug.cgi?id=471015">
+  <meta name="flags" content="paged">
+  <style>
+@page { size:5in 3in; margin:0.5in; }
+html,body {
+    color:black; background-color:white; font:0.5in/1 monospace; padding:0; margin:0;
+}
+fieldset {
+  padding: 0;
+  margin: 0;
+  border: 1em solid black;
+  border-bottom-color: white;
+}
+legend {
+  padding: 0;
+  margin: 0;
+  width: 3em;
+  height: 2em;
+}
+div { width:100%; height: 10em; border-bottom: 1em solid black; }
+x { display:block; height:1em; }
+</style>
+</head>
+<body>
+  <fieldset><x>A<br>B<br>C<br>D<br>E</x><legend><div></div></legend></fieldset>
+  <span>AFTER</span>
+</body>
+</html>